src/modules/Portal/components/AboutInfo/presenter.tsx

```diff
@@ -63,6 +65,9 @@ function AboutInfo(props: IAboutInfo) {
             Version { projectVersion }.{ buildNumber }.{ buildId }
           </p>
         }
+        <p {...classes('line', ['additional', 'padded'])}>
+          Vocabularies version: {vocabularyReleaseVersion}
```

Comment:
Probably to avoiding misunderstanding change label to `OMOP Vocabulary version`

Reply:
done